cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -127,16 +123,9 @@ public void execute(Project project) {
                 }
             }
 
-            List<Diagnostic> diagnostics = new ArrayList<>(emitResult.diagnostics().diagnostics());
-            if (!diagnostics.isEmpty()) {
-                //  TODO: When deprecating the lifecycle compiler plugin, we can remove this check for duplicates
-                //   in JBallerinaBackend diagnostics and the diagnostics added to EmitResult.
-                diagnostics = diagnostics.stream()
-                        .filter(diagnostic -> !jBallerinaBackend.diagnosticResult().diagnostics().contains(diagnostic))
-                        .collect(Collectors.toList());
-                if (!diagnostics.isEmpty()) {
-                    diagnostics.forEach(d -> out.println("\n" + d.toString()));
-                }
+            // Print diagnostics found during emit executable
+            if (!emitResult.diagnostics().diagnostics().isEmpty()) {
```

Comment:
We might have to do a null check for `emitResult.diagnostics()` right?

Reply:
According to the current implementation, `emitResult` would contain an arraylist even if it is empty. In that case, do we need a null check?